index.js

```diff
@@ -45,6 +45,7 @@ type QuestionPage implements Page {
   answers: [Answer]
   section: Section
   position: Int!
+  routingRuleSet: RoutingRuleSet!
```

Comment:
Should this be not nullable? Not all pages will have a rule set and any questionnaires created before routing wont be compatible with this schema?

Reply:
Yeah good shout. Probably worth making this nullable for pages that don't explicitly specify any routing rules.